.scripts/build.sh

```diff
@@ -4,6 +4,39 @@ php create-actions-filters-docs.php
 # Generate .pot file
 php -n $(which wp) i18n make-pot ../ ../languages/woocommerce-convertkit.pot
 
-# Build ZIP file
-rm ../convertkit-for-woocommerce.zip
-cd .. && zip -r convertkit-for-woocommerce.zip . -x "*.git*" -x ".scripts/*" -x ".wordpress-org/*" -x "tests/*" -x "vendor/*" -x "*.distignore" -x "*.env.*" -x "*codeception.*" -x "composer.json" -x "composer.lock" -x "*.md" -x "log.txt" -x "package-lock.json" -x "package.json" -x "phpcs.xml" -x "*.DS_Store"
\ No newline at end of file
+# Remove vendor directory
+cd ..
+rm -rf vendor
+
+# Run composer to only install non-dev dependencies
+composer install --no-dev
+
+# Build ZIP file, excluding non-Plugin files
+rm convertkit-for-woocommerce.zip
```

Comment:
Can we add `[ -e convertkit-for-woocommerce.zip ] && rm convertkit-for-woocommerce.zip` so no warning is printed if doesn't exist?